qpup_hw/config/qpup_hardware.yaml

```diff
@@ -1,106 +1,115 @@
 qpup:
   combined_robot_hw:
     shared_values:
-      splay_transmission_reduction: &shared_splay_transmission_reduction 40.0
-      hip_transmission_reduction: &shared_hip_transmission_reduction 40.0
-      knee_transmission_reduction: &shared_knee_transmission_reduction 64.61538461538462
-      left_knee_joint_offset: &shared_left_knee_joint_offset !degrees 30.0
-      right_knee_joint_offset: &shared_right_knee_joint_offset !degrees -30.0
+      left_splay_transmission_reduction: &shared_left_splay_transmission_reduction 40.0 # TODO
```

Comment:
why TODO? Also just double checking the 2pi is already in the robothw right

Reply:
because one needs to be negative

Reply:
2pi is in hw already